perfarce.py

```diff
@@ -479,42 +495,42 @@ def maxargs(self):
     def run(self, cmd, files=[], abort=True, client=None):
         'Run a P4 command and yield the objects returned'
 
-        c = ['p4', '-G']
+        c = [b'p4', b'-G']
         if self.server:
-            c.append('-p')
+            c.append(b'-p')
             c.append(self.server)
         if client or self.client:
-            c.append('-c')
+            c.append(b'-c')
             c.append(client or self.client)
         if self.root:
-            c.append('-d')
+            c.append(b'-d')
             c.append(shellquote(self.root))
 
         if files and len(files)>self.maxargs:
             tmp = TempFile('w')
             for f in files:
-                if self.ui.debugflag: self.ui.debug('> -x %s\n' % f)
-                print >>tmp.File,f
+                if self.ui.debugflag: self.ui.debug(b'> -x %s\n' % f)
+                print(f, file=tmp.File)
```

Comment:
python 3 only - from \_\_future\_\_ import print_function should work
or even tmp.File.write(f+'\n')